lib/board.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
```

Comment:
pry shouldn't be required in files in a PR (Like only require it during debugging)